http-server/src/test/java/org/triplea/server/lobby/game/listing/GameListingTest.java

```diff
@@ -108,48 +104,42 @@ void getGames() {
   final class KeepAlive {
     @Test
     void noGamesPresent() {
-      when(cache.getIfPresent(ID_0)).thenReturn(null);
-
       final boolean result = gameListing.keepAlive(API_KEY_0, GAME_ID_0);
       assertThat("Game not found, keep alive should return false", result, is(false));
 
-      verify(cache, never()).put(any(), any());
+      assertThat(cache.asMap(), is(Map.of()));
```

Comment:
Nit: Can use the empty map matcher here to be more explicit about the match criteria.